libs/ledger-live-common/src/reconciliation.ts

```diff
@@ -30,6 +30,7 @@ import {
   fromNFTRaw,
 } from "./account";
 import consoleWarnExpectToEqual from "./consoleWarnExpectToEqual";
+import { fromCardanoResourceRaw } from "./families/cardano/serialization";
```

Comment:
Please use the same pattern as the other families, imported from `"./account"` (see just above).